File: src/main/java/com/example/hotel_mvc/model/entity/Hotel.java
package com.example.hotel_mvc.model.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.time.Instant;
import java.util.Date;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "hotel")
public class Hotel {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    private Integer id;

    @Column(name = "name", length = 45)
    private String name;

    @Column(name = "name_furigana", length = 45)
    private String namefurigana;

    @Column(name = "url", length = 45)
    private String url;

    @Column(name = "trade_name", length = 45)
    private String tradename;

    @Column(name = "type", length = 45)
    private String type;

    @Column(name = "phone_number", length = 45)
    private String phonenumber;

    @Column(name = "fax_number", length = 45)
    private String faxnumber;

    @Column(name = "pref_code", length = 45)
    private String prefcode;

    @Column(name = "postal_code", length = 45)
    private String postalcode;

    @Column(name = "address", length = 45)
    private String address;

    @Column(name = "building_name", length = 45)
    private String buildingname;

    @Column(name = "license_number", length = 45)
    private String licensenumber;

    @Column(name = "image", length = 200)
    private String image;

    @Column(name = "payment_method", length = 45)
    private String paymentmethod;

    @Column(name = "create_at")
    private Date createat;

    @Column(name = "update_at")
    private Date updateat;

}

File: src/main/resources/static/js/hotel.js
const t = new Date();
const date = ('0' + t.getDate()).slice(-2);
const month = ('0' + (t.getMonth() + 1)).slice(-2);
const year = t.getFullYear();


const time = `${date}/${month}/${year}`;
$(document).ready(function () {

    $('.table .eBtn').on('click', function (event) {
        event.preventDefault();
        var href = $(this).attr('href');
        $.get(href, function (lsthotel, status) {
            $('.myModal #id').val(lsthotel.id);
            $('.myModal #name').val(lsthotel.name);
            $('.myModal #name_furigana').val(lsthotel.namefurigana);
            $('.myModal #url').val(lsthotel.url);
            $('.myModal #trade_name').val(lsthotel.tradename);
            $('.myModal #type').val(lsthotel.type);
            $('.myModal #phone_number').val(lsthotel.phonenumber);
            $('.myModal #fax_number').val(lsthotel.faxnumber);
            $('.myModal #pref_code').val(lsthotel.prefcode);
            $('.myModal #postal_code').val(lsthotel.postalcode);
            $('.myModal #address').val(lsthotel.address);
            $('.myModal #building_name').val(lsthotel.buildingname);
            $('.myModal #license_number').val(lsthotel.licensenumber);
            $('.myModal #image').val(lsthotel.image);
            $('.myModal #payment_method').val(lsthotel.paymentmethod);
            $('.myModal #create_at').val(lsthotel.createat);
            $('.myModal #update_at').val(time);
        })
        $('.myModal #exampleModal').modal();
    });
});